### ISSUE TITLE
failure rdmacm/test_md.sockaddr_accessibility/0 on hpc-test-node4

### ISSUE THREAD

**Issue by dmitrygladkov** [openucx/ucx#3751]
```
[2019-06-21T17:08:12.180Z] [ RUN      ] rdmacm/test_md.sockaddr_accessibility/0
[2019-06-21T17:08:12.180Z] [     INFO ] Testing ib0 with 65.65.65.1:0
[2019-06-21T17:08:12.738Z] /scrap/jenkins/workspace/ucx-10/contrib/../test/gtest/uct/test_md.cc:515: Failure
[2019-06-21T17:08:12.738Z] Value of: uct_md_is_sockaddr_accessible(md(), &sock_addr, UCT_SOCKADDR_ACC_LOCAL)
[2019-06-21T17:08:12.738Z]   Actual: false
[2019-06-21T17:08:12.738Z] Expected: true
[2019-06-21T17:08:12.738Z] [  FAILED  ] rdmacm/test_md.sockaddr_accessibility/0, where GetParam() = rdmacm (503 ms)
```
http://hpc-master.lab.mtl.com:8080/blue/rest/organizations/jenkins/pipelines/ucx/runs/396/nodes/104/steps/564/log/?start=0

**Comment by yosefe** [comment]
test passes after fixing fabric issues on the test node
```
[ RUN      ] rdmacm/test_md.sockaddr_accessibility/0
[     INFO ] Testing ib0 with 65.65.65.1:0
[     INFO ] Testing ib2 with 165.165.165.1:0
[     INFO ] Testing ib0 with fe80::ee0d:9a03:92:a04e:0
[     INFO ] Testing ib2 with fe80::ee0d:9a03:92:a054:0
[       OK ] rdmacm/test_md.sockaddr_accessibility/0 (4 ms)
[----------] 1 test from rdmacm/test_md (4 ms total)
```
